libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -401,16 +401,16 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
           />
           <Button buttonText="Run" onClick={() => run(state.data, state.source, state.file)} disabled={(state.data === null || categoryIndex.length === 0) && !slitherEnabled }/>
         </div>
-        <div className="d-flex" id="enableSlitherAnalysis">
+        <div className="d-flex" id="enableSlitherAnalysis" style={{ visibility: showSlither } as CSSProperties }>
           <RemixUiCheckbox
             id="enableSlither"
             inputType="checkbox"
             onClick={handleSlitherEnabled}
             checked={slitherEnabled}
             label="Enable Slither Analysis"
             onChange={() => {}}
-            visibility = {showSlither}
           />
+          <a href="https://remix-ide.readthedocs.io/en/latest/slither.html#enable-slither-analysis" target="_blank"><i className="ml-2 fas fa-info" title="Know how to use Slither Analysis"></i></a>
```

Comment:
small notes from me:
ml-3 looks better to me.
I would not put two tags at the same line

Reply:
ok